executor/builder.go

```diff
@@ -2040,6 +2043,94 @@ func (b *executorBuilder) buildIndexLookUpReader(v *plannercore.PhysicalIndexLoo
 	return ret
 }
 
+func buildNoRangeIndexMergeReader(b *executorBuilder, v *plannercore.PhysicalIndexMergeReader) (*IndexMergeReaderExecutor, error) {
+	partialPlanCount := len(v.PartialPlans)
+	partialReqs := make([]*tipb.DAGRequest, 0, partialPlanCount)
+	partialStreamings := make([]bool, 0, partialPlanCount)
+	indexes := make([]*model.IndexInfo, 0, partialPlanCount)
+	keepOrders := make([]bool, 0, partialPlanCount)
+	descs := make([]bool, 0, partialPlanCount)
+	for i := 0; i < partialPlanCount; i++ {
+		tempReq, tempStreaming, err := b.constructDAGReq(v.PartialPlans[i])
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+		if is, ok := v.PartialPlans[i][0].(*plannercore.PhysicalIndexScan); ok {
+			keepOrders = append(keepOrders, is.KeepOrder)
+			descs = append(descs, is.Desc)
+			tempReq.OutputOffsets = []uint32{uint32(len(is.Index.Columns))}
```

Comment:
I do not understand this line. I have 2 questions:
1. Do we will only fetch one column from the IndexScan?
2. What does it mean that the `OutputOffsets` is `len(is.Index.Columns)`?